releases.md

```diff
@@ -0,0 +1,195 @@
+# Express Release Process
+
+This document contains the technical aspects of the Express release process. The
+intended audience is those who have been authorized by the Express Technical
+Committee (TC) to create, promote and sign official release builds for Express,
+as npm packages hosted on https://npmjs.com/package/express.
+
+## Who can make releases?
+
+Release authorization is given by the Express TC. Once authorized, an individual
+must have the following access permissions:
+
+### 1. Github release access
+
+The individual making the release will need to be a member of the
+expressjs/express team with Write or Admin permission level so they are
+able to tag the release commit and push changes to the expressjs/express
+repository (see Steps 4 and 5).
+
+[tunniclm: Is there  a separate way to indicate whether an individual is allowed
+           to tag a release, or some higher level of permission?]
+
+### 2. npmjs.com release access
+
+The individual making the release will need to be a member of the expressjs
+organisation on npmjs.com so they are able to publish the release package
+(see Step 6).
```

Comment:
> Yeah not sure an npm org would work but a shared ID for publishing releases for express related stuff would be good. The shared secrets can be managed in a private repo the way we do shared secrets for core stuff. (There may
already be some tool for this somewhere)

What is the purpose of that? `npm` already provide a mechanism for multiple people to have access, and it's very similar to a GitHub repository: you simply add people to the `npm` package, for example `npm owner add <pkg> <username>` (documentation https://docs.npmjs.com/cli/owner). Are there specific concerns with using this approach?

> Maybe an org would be possible for the future since there are lots of packages besides express itself to manage.

Yea, this would be ideal, but as brought up before, as far as I know, an org can only publish modules in the org's scope, not global modules. Express and all it's other modules are all global modules and not scoped modules.